common/types.go

```diff
@@ -43,6 +43,9 @@ var (
 	// ErrNotJSON is returned when a response is not JSON.
 	ErrNotJSON = errors.New("response is not JSON")
 
+	// ErrNullJSON is returned when a JSON key is null.
+	ErrNullJSON = errors.New("value of JSON key is null")
```

Comment:
this isn't used anywhere

Reply:
It is used here. Which in turn is used by `GetString, GetInteger` methods
https://github.com/amp-labs/connectors/blob/d6481dd91feba64de85faaab0105eeb917b3f564/common/parse.go#L278-L284